templates/annotate.sh

```diff
@@ -124,6 +127,72 @@ vep () {
   !{singularity_vep} vep "${args[@]}"
 }
 
+capice () {
+  local vcfPath="!{vcfPath}"
+  local vep_args=()
+  vep_args+=("--input_file" "${vcfPath}")
+  vep_args+=("--format" "vcf")
+  vep_args+=("--output_file" "${vcfPath%%.*}_prepared.vcf.gz")
```

Comment:
I don't like this `${...%%.*}` magic. What is the purpose?

Reply:
It strips the `.vcf.gz` extension from the path so the files in this process won't look like this: `filename.vcf.gz.capice.annotated.tsv.vcf.gz` as I'm afraid it might break because the weird extensions.